src/assets/locales/fr.json

```diff
@@ -337,23 +382,28 @@
   "preferences-screen.material-quality-setting.medium": "Medium",
   "preferences-screen.preference.allow-multiple-hubs-instances": "Désactiver la sortie automatique quand de multiples instances de hubs sont ouvertes",
   "preferences-screen.preference.animate-waypoint-transitions": "Animer les transitions entre les waypoints",
+  "preferences-screen.preference.audio-clipping-threshold": "Seuil pour l'audio clipping",
   "preferences-screen.preference.disable-auto-gain-control": "Désactiver le contrôle automatique du gain du micro",
   "preferences-screen.preference.disable-auto-pixel-ratio": "Désactiver les ajustements automatiques du ratio de pixels",
   "preferences-screen.preference.disable-backwards-movement": "Désactiver les mouvements en marche arrière",
   "preferences-screen.preference.disable-echo-cancellation": "Désactiver l'annulation d'écho du micro",
   "preferences-screen.preference.disable-idle-detection": "Désactiver la sortie automatique si inactif ou en arrière-plan",
   "preferences-screen.preference.disable-movement": "Désactiver le mouvement",
   "preferences-screen.preference.disable-noise-suppression": "Désactiver la suppression de bruit du micro",
+  "preferences-screen.preference.disable-panning": "Désactiver le panning audio gauche/droite",
   "preferences-screen.preference.disable-sound-effects": "Désactiver les effets sonores",
   "preferences-screen.preference.disable-strafing": "Désactiver le mitraillage",
   "preferences-screen.preference.disable-teleporter": "Désactiver la téléportation",
+  "preferences-screen.preference.enable-audio-clipping": "Activer l'audio clipping",
   "preferences-screen.preference.enable-dynamic-shadows": "Activer les ombres dynamiques",
   "preferences-screen.preference.enable-gyro": "Activer le gyroscope (si compatible avec le navigateur / l'appareil)",
   "preferences-screen.preference.enable-on-screen-joystick-left": "Activer le joystick sur l'écran à gauche pour se déplacer",
   "preferences-screen.preference.enable-on-screen-joystick-right": "Activer le joystick sur l'écran à droite pour se déplacer",
+  "preferences-screen.preference.fast-room-switching": "Activer le basculement rapide de salle",
   "preferences-screen.preference.global-media-volume": "Volume audio des médias",
   "preferences-screen.preference.global-voice-volume": "Volume audio de la voix entrante",
   "preferences-screen.preference.invert-touchscreen-camera-move": "Inverser la direction du mouvement de caméra pour les écrans tactiles",
+  "preferences-screen.preference.lazy-load-scene-media": "Activer le chargement différé des ressources média de la scène",
```

Comment:
"preferences-screen.preference.lazy-load-scene-media": "Enable Scene Media Lazy Loading",

Reply:
"Activer le chargement asynchrone des ressources multimédia de la scène" ?

Reply:
I'm ok with your proposal. I have a doubt on "asynchrone" though, "différé" is in my opinion better understandable for the user?
I changed it in https://github.com/mozilla/hubs/pull/5002/commits/149b1fc31dfc8e9eacdd3487e95874e6697a8fad
If there is another opinion on that, please comment.